docs/image_assertions.rst

```diff
@@ -0,0 +1,117 @@
+Image Assertions
+================
+
+The bok-choy framework uses `Needle`_ to provide the ability to capture portions of a rendered page
+in the browser and assert that the image captured matches that of a baseline.
+
+The general methodology for creating a test with a screenshot assertion is:
+
+* `Write your page object and testcase code to navigate the system under test`_
+* `Add the call to assertScreenshot`_
+* `Create the initial baseline screenshot`_
+* `Execute your testcases after changes to the system under test`_
+* `Advanced features`_
+
+Write your page object and testcase code to navigate the system under test
+--------------------------------------------------------------------------
+If you are not familiar with how to write a bok-choy page object and testcase, first check out the Tutorial.
+Here is an example of a test that navigates to the edx.org home page.
+
+**page.py**, which contains the page object:
+
+.. code-block:: python
+
+    from bok_choy.page_object import PageObject
+
+    class EdxHomePage(PageObject):
+        url = 'http://www.edx.org'
+
+        def is_browser_on_page(self):
+            return 'edX' in self.browser.title.lower()
+
+**my_test.py**, which contains the test code:
+
+.. code-block:: python
+
+    from bok_choy.web_app_test import WebAppTest
+    from page import EdxHomePage
+
+    class TestEdxHomePage(WebAppTest):
+
+        def test_page_existence(self):
+            EdxHomePage(self.browser).visit()
+
+Add the call to assertScreenshot
+--------------------------------
+assertScreenshot() take two arguments: a CSS selector for the element to capture and a filename for the image.
+Here is the same testcase, with an assertion added that will ensure that the site logo for the edx.org home
+page has not changed.
+
+* The first argument, "img.site-logo" is the css locator for the element that we want to capture and compare.
+* The second argument, "edx_logo_header" is the filename (.png will be appended automatically) that will be used
```

Comment:
Does that mean you can only use .png files? Should we point that out? (I realize you have to take a screenshot with the tool, but....)